definitions/tools/homer_tag_directory.cwl

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: 'Creating the homer tag directory'
+baseCommand: ["/bin/bash", "/gscmnt/gc2708/info/medseq/apl/chipseq/src/bam2homer.sh"]
```

Comment:
Probably a bad idea to link to files on the filesystem.  It'd be better to copy the script into the homer container, or just add it here as an inline bash script.